x-pack/platform/plugins/shared/alerting/server/saved_objects/model_versions/rule_model_versions.ts

```diff
@@ -50,4 +51,11 @@ export const ruleModelVersions: SavedObjectsModelVersionMap = {
       create: rawRuleSchemaV5,
     },
   },
+  '6': {
+    changes: [],
```

Comment:
I think we should index the investigation guide blob to be able to search rules by the investigation guide. You would need to do:

```
changes: [
    {
      type: 'mappings_addition',
      addedMappings: {
        investigation_guide: {
          properties: {
            blob: {
              type: 'string',
            },
          },
        },
      },
    },
  ],
```

Wdyt?

Reply:
That sounds good - I do not think `string` is an accepted value for `type` though, should we use `text` instead?

Reply:
Yes, my bad 🙂. I meant [text](https://www.elastic.co/docs/reference/elasticsearch/mapping-reference/text).

Reply:
You can check 592fba02c3636f4fca9dc2589697303cce131c79 where I have implemented this, but also nested `investigation_guide` under `artifacts` as we discussed offline.